lib/model/query/users.js

```diff
@@ -46,6 +46,9 @@ invalidatePassword.audit = (user) => (log) => log('user.update', user.actor, {
   data: { password: null }
 });
 
+const updateLastLoginAt = (user) => ({ run }) =>
+  run(sql`UPDATE users SET "lastLoginAt"=clock_timestamp() WHERE "actorId"=${user.actor.id}`);
```

Comment:
I think it makes sense and is consistent with other usage that you're using `clock_timestamp()` here. :+1: I just wanted to note that it could end up being slightly different from the timestamp of the `user.session.create` event. Presumably not by much, but maybe there could be an awkward edge case involving rounding where the two timestamps are displayed as happening in different seconds. I don't think we need to change anything. For example, I don't think we need to sequence the `UPDATE` after the audit log `INSERT` in order to reuse the audit log timestamp or anything like that. Mostly I just wanted to mention it as a case to consider as we all continue discussing getodk/central#1216.